en/zbMATH.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Fetching entries from zbMATH
+helpCategories: ["Fetching entries from the web", "... using online bibliographic database"]
+
+---
+
+# Fetching entries from zbMATH
+
+[zbMATH](https://www.zbmath.org/) is a reviewing service providing reviews and abstracts for articles in pure and applied mathematics ([Wikipedia](https://en.wikipedia.org/wiki/Zentralblatt_MATH)).
+
+
+To fetch entries from zbMATH, choose **Search -&gt; Web search**, and the search interface will appear in the side pane.
+Select **zbMATH** in the dropdown menu.
+To start a search, enter the words of your query, and press <kdb>RETURN</kdb> or the **Fetch** button.
```

Comment:
Please use `Return` instead of `RETURN`. Only write the first letter in upper case letter. Oh, I think, we use `Enter` instead of `Return`.